src/ng-jhipster.module.ts

```diff
@@ -44,19 +44,6 @@ export function missingTranslationHandler(configService: JhiConfigService) {
 
 @NgModule({
     imports: [
-        CommonModule,
```

Comment:
I think we should also move `translatePartialLoader` to generator to remove build dependency on `process.env.BUILD_TIMESTAMP` and resource bundles path dependency on `i18n/`. Otherwise, we can pass these inputs via config. WDYT?

Reply:
I don't understand the advantage of moving this into the generator. I think the build timestamp is there to do cache busting if I'm correct, so what would we gain if we move that into the generator ?

Reply:
I was thinking about making it independent of current build setup (e.g. to use in `angular-cli` builds). On further thought, users can now override this method in the generated code, so, this should be fine.